data_centre/data.py

```diff
@@ -7,6 +7,24 @@
 from itertools import cycle
 from omxplayer.player import OMXPlayer
 from shutil import copyfile
+import threading 
+
+from data_centre import plugin_collection
+
+class AsyncWrite(threading.Thread):
```

Comment:
@doctea  looks good - what do we need AsyncWrite for though ?

Reply:
> @doctea looks good - what do we need AsyncWrite for though ?

Some preset json (eg automation clips) can be pretty large (1mb+) and it lags when encoding/saving otherwise -- the ShaderLoopRecordPlugin records every time you press 'stop recording' so it was intended to keep things running smooth.